mods/ts/rules/aircraft.yaml

```diff
@@ -35,6 +35,50 @@ DPOD:
 		EmptySequence: pip-ammoempty
 		Palette: pips
 
+DPOD2:
+	Inherits@2: ^ExistsInWorld
+	Inherits@CARGOPIPS: ^CargoPips
+	Valued:
+		Cost: 10
+	Tooltip:
+		Name: Drop Pod
+	Health:
+		HP: 6000
+	Armor:
+		Type: Light
+	Aircraft:
+		TurnSpeed: 5
+		Speed: 300
+		CruiseAltitude: 8000
+		MaximumPitch: 110
+		LandableTerrainTypes: Clear, Road, Rail, DirtRoad, Rough
+	HiddenUnderFog:
+		Type: CenterPosition
+	BodyOrientation:
+		UseClassicPerspectiveFudge: False
+	RenderSprites:
+		Image: pod
+	WithFacingSpriteBody:
+	QuantizeFacingsFromSequence:
+	HitShape:
+	Cargo:
+		Types: Infantry
+		MaxWeight: 1
+		UnloadVoice: Move
+		EjectOnDeath: true
```

Comment:
Having this and `SpawnActorOnDeath` looks a bit duplicated. Why do we need both?

Reply:
We also have `EjectOnDeath`. They all have a slightly different use case.

Reply:
You misunderstood the question. I wasn't asking in general but why *this unit* needs both.

Reply:
You are right. That is copy and paste leftovers. Removed it.